rocks.thorium.Mercury.yml

```diff
@@ -0,0 +1,107 @@
+id: rocks.thorium.Mercury
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    version: '23.08'
+    directory: lib/ffmpeg
+    add-ld-path: .
+cleanup-commands:
+  - mkdir -p $FLATPAK_DEST/lib/ffmpeg
+command: mercury-browser
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pcsc
+  - --socket=pulseaudio
+  - --socket=cups
+  - --persist=.mercury
+  - --filesystem=xdg-download:rw
+  - --filesystem=xdg-run/pipewire-0
+  - --device=dri
+  - --talk-name=org.freedesktop.FileManager1
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --talk-name=org.a11y.Bus
+  - --talk-name=org.gnome.SessionManager
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.gtk.vfs.*
+  - --talk-name=org.freedesktop.Notifications
+  - --env=MOZ_ENABLE_WAYLAND=1
```

Comment:
Isn't this the default now? If it's not, no need to force it

Reply:
I removed this.